src/main/resources/application.yaml

```diff
@@ -98,11 +98,11 @@ queues:
     connection-string: ${QUEUE_PROCESSED_ENVELOPES_READ}
     queue-name: ${QUEUE_PROCESSED_ENVELOPES_NAME}
   notifications:
-    access-key: ${QUEUE_NOTIFICATIONS_SEND_ACCESS_KEY}
+    access-key: ${QUEUE_NOTIFICATIONS_SEND_ACCESS_KEY:}
```

Comment:
do we need the default value here?

Reply:
same for `namespace-override`

Reply:
we do as notification is not yet merged - it lives in `reform-scan` domain, not `bulk-scan`